src/main/webapp/js/userMap.js

```diff
@@ -1,18 +1,17 @@
-document.addEventListener('DOMContentLoaded', function(event) {
+function handleData(err, userData) {
     // based on example from https://github.com/markmarkoh/datamaps/blob/master/src/examples/highmaps_world.html
     // Country code: https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3
-
+    if (err) {
+        console.log('error in getting data');
```

Comment:
This is hardly helpful. Should show the error to the user (and ask them to reload the page) via status message or alike.